### PR TITLE
Upgrade to VS 2017

### PR DESCRIPTION
**Pull request by natemcmaster**
Remove project.json and move to MSBuild.

Had to disable standalone tests because of dotnet/sdk#696. I will open an issue to re-enable them.

Resolves https://github.com/aspnet/IISIntegration/issues/321

### CODE REVIEW

File: test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/NtlmAuthentationTest.cs [hunk 1]
```diff
@@ -11,35 +11,51 @@
 using Microsoft.Extensions.Logging;
 using Xunit;
 using Xunit.Sdk;
+using Xunit.Abstractions;
+using Microsoft.AspNetCore.Testing;
```
Comment (BrennanConroy) [inline test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/NtlmAuthentationTest.cs]: Sort

File: test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/NtlmAuthentationTest.cs [hunk 2]
```diff
@@ -11,35 +11,51 @@
 using Microsoft.Extensions.Logging;
 using Xunit;
 using Xunit.Sdk;
```
Comment (Tratcher) [inline test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/NtlmAuthentationTest.cs]: sort

File: test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/XunitLogger.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+// See License.txt in the project root for license information
+
+using System;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    public class XunitLogger : ILogger, IDisposable
```
Comment (Tratcher) [inline test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/XunitLogger.cs]: Should we add this to Testing?

**Comment by BrennanConroy** [comment]
Disabling is a little different from removing the Standalone project entirely. Shouldn't it still be "ported"

File: test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/XunitLogger.cs (hunk 3 above)
Comment (natemcmaster) [inline test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/XunitLogger.cs]: Yes. https://github.com/aspnet/Testing/issues/243

**Comment by natemcmaster** [comment]
We don't need separate standalone projects anymore. Standalone vs portable is a publish time decision, not a build decision.

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,24 @@
+﻿# EditorConfig is awesome:http://EditorConfig.org
```
Comment (Tratcher) [inline .editorconfig]: @muratg is this something we want in all the projects?

@natemcmaster Which editors understand this? Does it require plugins?

Comment (natemcmaster) [inline .editorconfig]: > Which editors understand this?

VS 2017 (no plugins). 
VS Code with plugin.

It's not yet enforced by csc, but IIUC they are planning to add that.

**Pull request merged**